.github/ISSUE_TEMPLATE/comp_contrib_instrumentation_awslambda.md

```diff
@@ -1,10 +1,10 @@
 ---
-name: OpenTelemetry.Contrib.Instrumentation.AWSLambda
-about: Issue with OpenTelemetry.Contrib.Instrumentation.AWSLambda
+name: OpenTelemetry.Instrumentation.AWSLambda
+about: Issue with OpenTelemetry.Instrumentation.AWSLambda
 labels: comp:contrib.instrumentation.awslambda
```

Comment:
See https://github.com/open-telemetry/opentelemetry-dotnet-contrib/pull/593#issuecomment-1228237203:

If we want to change this to

```suggestion
labels: comp:instrumentation.awslambda
```

then a maintainer should rename this in the GitHub repository settings first.